packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -537,7 +537,14 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
                             }
                         )
                     }
-                    latch.await()
+                    try {
+                        latch.await()
+                    } catch (e: InterruptedException) {
```

Comment:
Is this the only exception we care about? Should this be `catch (e: Exception)`?

Reply:
Yea `await` throws only `InterruptedException`.